modules/admin_manual/pages/installation/ubuntu_18_04.adoc

```diff
@@ -160,7 +160,7 @@ occ config:system:set trusted_domains 1 --value="$myip"
 
 [source,console,subs="attributes+"]
 ----
-echo "*/15  *  *  *  * /usr/bin/php -f {install-directory}/cron.php" \
+echo "*/15  *  *  *  * /usr/bin/php occ system:cron" \
```

Comment:
Whenever there is a cron job to be entered, you need to ensure that the full path to the occ command is used.
Needs checking for all changes with regards to cron.
Beside the necessity of having the complete path to make it run successfully, imagine you have more than one instance on different domains on the same server.

Reply:
Thanks, @mmattel

Reply:
All references have been updated.